backend/tests/conftest.py

```diff
@@ -25,3 +25,193 @@ def raise_http_status_error_501(_):
         request=Request("get", "http://"),
         response=Response(501),
     )
+
+
+covalent_transfers_v2_transfers = [
```

Comment:
Is this used?

Reply:
Also, in terms of making tests for the Covalent adapter, as it should be moved into the `treasury` domain-module, I suggest we also move tests in `app/treasury/tests`.

That's the path I started to take in `app/spread/tests` and it seems to me more relevant for code discovery of unit tests. Then the root `backend/tests` folder should be used for integration tests (or maybe libraries/utilities which don't have a clear module).

Reply:
> Is this used?

If you're referring to `covalent_transfers_v2_transfers` then yes ([starting from here](https://github.com/butterdao/whip/blob/refractor/provider_error_handling/backend/tests/http/test_dataproviders.py#L171)).